pandas/tests/test_categorical.py

```diff
@@ -1554,18 +1554,6 @@ def test_deprecated_labels(self):
             res = cat.labels
         self.assert_numpy_array_equal(res, exp)
 
-    def test_deprecated_levels(self):
```

Comment:
If that's the case, then we *do* need to keep this, in order to test reading an older pickle (and converting to the new format). That really *should* have been in the generate script, but maybe not.